platform/admin/ui/src/main/webapp/js/views/configuration/ConfigurationEdit.view.js

```diff
@@ -269,15 +269,6 @@ define([
                 if (!this.model.get('properties').has('service.pid')) {
                     this.model.get('properties').set('service.pid', this.service.get('id'));
                 }
-
```

Comment:
Deleting all the code below makes the `if` at line 268 step into another `if`. We could probably just combine the two conditions with `&&`